src/zero/zero.clj

```diff
@@ -58,3 +58,6 @@
               str/trim
               edn/read-string)
       {:version "SOME BOGUS VERSION"}))
+
+
+(get-the-version)
```

Comment:
Why is this added here?

Reply:
Looks like debugging cruft.

Reply:
Yeah, that should be removed from here... will do